src/staging/staging_main.py

```diff
@@ -250,14 +243,49 @@ def run_staging(
     cellno_df = read_csv(cellno_path)
     StagingMainLogger.info("Covarage File Loaded Successfully...")
 
-    # Loading PG numeric to alpha mapper
-    StagingMainLogger.info("Loading PG numeric to alpha File...")
+    # Loading PG alpha to numeric mapper - possibly, deprecated
+    StagingMainLogger.info("Loading PG alpha to numeric File...")
     pg_alpha_num_path = paths["pg_alpha_num_path"]
     check_file_exists(pg_alpha_num_path)
     pg_alpha_num = read_csv(pg_alpha_num_path)
     val.validate_data_with_schema(pg_alpha_num, "./config/pg_alpha_num_schema.toml")
+    pg_alpha_num = val.validate_many_to_one(
```

Comment:
Where does `pg_alpha_num` get used?

Reply:
in tau and gb sas outputs. The requirement is to produce a column called "selgrpno", which is PG in numeric format.  In join_pg_numeric function, we create a column 201_numeric. In tau.toml, we create [selgrpno] old_name = "201_numeric"

Potentially, the BaU may not need selgrpno, but they haven't confirmed it.

Finally,  for short forms the correct way to create "selgrpno" is to use sic to pg_numeric mapper. I have not built it, because it's not clear if they need it, and maybe not required at all.